colors/dracula.vim

```diff
@@ -431,15 +433,16 @@ hi! link rubyGlobalVariable DraculaPurple
 hi! link rubyRegexpDelimiter DraculaRed
 
 "}}}2
-" YAML: {{{2
+" Sass: {{{2
 
-hi! link yamlBlockMappingKey DraculaCyan
-hi! link yamlPlainScalar DraculaYellow
-hi! link yamlAnchor DraculaPinkItalic
-hi! link yamlAlias DraculaGreenItalicUnderline
-hi! link yamlNodeTag DraculaPink
-hi! link yamlFlowCollection DraculaPink
-hi! link yamlFlowIndicator Delimiter
+hi! link sassProperty cssProp
+hi! link sassMixinName Function
+hi! link sassClass DraculaGreen
+hi! link sassClassChar DraculaGreen
+hi! link sassInterpolationDelimiter DraculaPink
```

Comment:
Is there a reason not to link to the `Delimiter` group? Although it appears to be a different color...

Reply:
Not sure I fully understand the question here. I'm assuming you're asking why we don't do `hi! link Delimiter DraculaPink` rather than `hi! link sassInterpolationDelimiter DraculaPink`?

The reason for that is because not every syntax uses the term `Delimiter` correctly. So this would lead to discrepancies. 

This answer applies also to your comment below.

Reply:
Actually I meant the opposite: why not `hi! link sassInterpolationDelimiter Delimiter` ?

Reply:
See 4596686 for a small last-minute adjustment that I made that changes the new sass tweaks so that instead of pointing to `Draculaxxx`, it instead links to the css equivalent, if available.

Gonna go ahead and merge since I didn't add anything beyond that.